### ISSUE TITLE
Z movement is awful

### ISSUE THREAD

**Issue by Krigeris** [pa1nki113r/Project_Brutality#242]
If its capable of emulating how doom feels, the default settings aren't it, I cannot get the movement to feel like doom either with tweaking. The speed, the acceleration, the jump height, jump speed, jump control its all needlessly different then what we've been playing with for years. I honestly just preferred the old double jump/climb and movement, if people wanted higher customization they would add z-movement themselves I reckon.

Basically; I think PB should be z-move compatible not z-move integrated, unless you can get the default settings to feel exactly like the game feels without it (but with pb additions like climbing/double jump).

**Comment by Krigeris** [comment]
small note wish I could edit the title, z movement is not awful but it _feels_ awful when I want the standard movement as the default and to modify it from there.

**Comment by JMartinez9820** [comment]
Have you tried PA1N's settings? This is what he recommends
https://cdn.discordapp.com/attachments/628066836751319060/628402950120931358/unknown.png
https://cdn.discordapp.com/attachments/628066836751319060/628402844957016094/unknown.png

**Comment by Krigeris** [comment]
Those settings certainly feel better but definitely don't feel like doom at all. Still the acceleration friction, and especially jumping is off. In its current state I don't even want to play this unfortunately, Im about to go grab an older version of PB and play that instead. Old PB pivot double jumping was amazing and it still feels like the same engine I'm used too.

**Comment by JMartinez9820** [comment]
That's unfortunate. The movement Pa1n suggest was based of Quake/Dusk anyway. Feel free to close this issue if necessary.

**Comment by Krigeris** [comment]
I feel like this is a huge issue as movement is kinda one of the defining features of doom, is the team dead set on using z-movement? Is it possible to emulate standard doom movement in it as the default?

Idk I would leave it open for debate, or until someone answers the above, unless this isn't the place for that.

**Comment by scaryred24** [comment]
Its still being tooled around as zmovement also gets updated. Unless you can find a better setting that will best fit your play style as for what you feel is what it should be, then unfortunately no pain is deadset in keeping zmovement at the current state of things.

**Comment by colegmr** [comment]
I agree with Krigeris zmove seems to cause more problems then its worth
like trying to fix something that isn't broken
though it is impressive how much its been improved since implementation.

**Comment by goldenmob72** [comment]
It's not THAT bad

**Comment by tunbridgep** [comment]
I agree it feels weird and I sort of don't like it, although I am used to it by now.

I think we should make it a priority to make ZMove feel as close as possible to the original doom, and set those as it's default settings

**Comment by caviidae** [comment]
`Top Running Speed` needs to be `15.0` to be able to make some secrets

**Comment by tunbridgep** [comment]
The big problem isn't necessarily the running speed, it's the accelleration/decelleration. The doomguy speeds up to full speed and slows down way too quickly and it makes it harder to reach some things where you need to "fly" more in the air, or you need to move in certain ways

**Comment by JMartinez9820** [comment]
According to pa1nki113r
> Going to be updating zmovement to the latest version soon

**Comment by goldenmobster** [comment]
It's already been updated

**Comment by scaryred24** [comment]
Locking this issue before any negative backlash is sent this way.
